### PR TITLE
test(axe): re-implement the accessibility tests

### PR DESCRIPTION
**Pull request by DlgSHi**
### Proposed behaviour

- Decreased the `retry` package tries from `3` to `1`;
- Re-implement the `accessibility` tests, now we are using `axe-core`, testing `WCAG 2.0` and `WCAG 2.1` instead of comparing amount of `violations`; 
(in this case we speed up `accessibility` suite:
`accessibilityCommon`: `2:23m` -> `0:33s`
`accessibilityCommonSecondPart`: `2:06m` -> `0:35s`
`accessibilityExperimental`: `1:21m` -> `0:19s`
`accessibilityDesignSystem`: `1:17m` -> `0:20s`
`accessibilityTest`: `0:21s` -> `0:06s`)
- Add new `icon` `visual` story, where we have rendered `all` icons in all `shapes` / `sizes` and 1 icon in all `states`;
- Removed the last part of `applitools`;
- Removed unnecessary `env's` from `cypress.env.json` file.

### Current behaviour

We need to speed up regression suite. 

### Checklist

- [ ] Screenshots are included in the PR
- [ ] Carbon implementation and Design System documentation are congruent
- [ ] All themes are supported
- [x] Commits follow our style guide
- [ ] Unit tests added or updated
- [x] Cypress automation tests added or updated
- [ ] Storybook added or updated
- [ ] Typescript `d.ts` file added or updated

### CODE REVIEW

File: cypress/features/regression/accessibility/accessibilityCommon.feature [hunk 1]
```diff
@@ -13,15 +12,13 @@ Feature: Accessibility tests
 
   @accessibility
   Scenario: Component button as sibling
-    Given I open "button" component page as sibling in no iframe
-    When I open Accessibility Tab
+    When I open "button" component page as sibling in noIFrame
     Then "button" component has no violations in Accessibility section
 
-  @accessibility
+  @ignore
```
Comment (KatarzynaQA) [inline cypress/features/regression/accessibility/accessibilityCommon.feature:18]: why is it ignored ?

Comment (DlgSHi) [reply cypress/features/regression/accessibility/accessibilityCommon.feature:18]: updated

File: cypress/support/step-definitions/accessibility-steps.js [hunk 2]
```diff
@@ -1,17 +1,14 @@
-import { violations } from '../../locators/accessibility';
-import { clickAccessebilityTab } from '../helper';
-import { reRunTestsButton } from '../../locators';
+const A11YOptions = {
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/support/step-definitions/accessibility-steps.js:1]: updated

File: cypress/support/step-definitions/accessibility-steps.js [hunk 3]
```diff
@@ -1,17 +1,14 @@
...
+  runOnly: {
+    type: 'tag',
+    values: [
+      'wcag2aa', // WCAG 2.0 Level AA
+      'wcag21aa', // WCAG 2.1 Level AA
+    ],
+  },
+};
 
 Then('{string} component has no violations in Accessibility section', () => {
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/support/step-definitions/accessibility-steps.js]: updated

File: .storybook/preview.js [hunk 4]
```diff
@@ -34,7 +34,19 @@ addParameters({
...
-  chromatic: { disable: false }
+  chromatic: { disable: false },
+  parameters: {
+    a11y: {
+      // axe-core optionsParameter (https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#options-parameter)
+      options: {
+        runOnly: {
+          type: 'tag',
+          values: [
+            'wcag2aa', // WCAG 2.0 Level AA
```
[the remark this answers is not in the document]
Comment (jamime) [reply .storybook/preview.js:45]: I created an issue for this https://github.com/dequelabs/axe-core/issues/2327

File: .storybook/preview.js [hunk 5]
```diff
@@ -34,7 +34,19 @@ addParameters({
       return a[1].id.localeCompare(b[1].id);
     }
   },
-  chromatic: { disable: false }
+  chromatic: { disable: false },
+  parameters: {
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply .storybook/preview.js]: fixed

File: package.json [hunk 6]
```diff
@@ -130,11 +130,13 @@
     "xhr-mock": "git://github.com/resin-io-modules/xhr-mock.git#improvements"
   },
   "dependencies": {
+    "axe-core": "^3.5.5",
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply package.json]: updated

File: package.json [hunk 7]
```diff
@@ -130,11 +130,13 @@
...
     "axios": "^0.19.0",
     "bignumber.js": "^9.0.0",
     "bowser": "~1.5.0",
     "classnames": "~2.2.3",
     "crypto-js": "~3.1.6",
+    "cypress-axe": "^0.8.1",
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply package.json]: updated

**Pull request merged**